prime-router/metadata/fhir_mapping/hl7/resource/DiagnosticReport.yml

```diff
@@ -59,6 +59,16 @@ identifier_4:
     code: "PGN"
     display: "Placer Group Number"
 
+identifier_5:
+  condition: $type NOT_NULL
+  valueOf: datatype/Identifier
+  generateList: true
+  expressionType: resource
+  vars:
+    type: OBR.26.1
```

Comment:
This type seems wrong.  I am guessing it needs to have an extra number: OBR.26.1.x

Reply:
type pulls out the sub elements in the field!